src/services/predictService.ts

```diff
@@ -0,0 +1,81 @@
+import _ from "lodash";
+import url from 'url';
+import { constants } from "../common/constants";
+import { interpolate, strings } from "../common/strings";
+import { AppError, ErrorCode, IProject } from "../models/applicationState";
+import ServiceHelper from "./serviceHelper";
+
+export enum AutoLabelingStatus {
+    none,
+    running,
+    done
+}
+export class PredictService {
+
+    constructor(private project: IProject) {
```

Comment:
remove constructor if it doesn't initialize anything

Reply:
the constructor will declare private property named project

Reply:
the constructor is is required, it initialize the project proerty, the project is used in methods in PredictService